test/test_migrate.py

```diff
@@ -174,14 +174,30 @@ def test_crypto():
         from libqtile import bar
         from libqtile.widget import BitcoinTicker
 
-        bar.Bar([BitcoinTicker()], 30)
+        bar.Bar(
+            [
+                BitcoinTicker(crypto='BTC', format='BTC: {avg}'),
+                BitcoinTicker(format='{crypto}: {avg}', font='sans'),
+                BitcoinTicker(),
+                BitcoinTicker(currency='EUR', format='{avg}', foreground='ffffff'),
+            ],
+            30
+        )
     """)
 
     expected = textwrap.dedent("""
         from libqtile import bar
         from libqtile.widget import CryptoTicker
 
-        bar.Bar([CryptoTicker()], 30)
+        bar.Bar(
+            [
+                CryptoTicker(crypto='BTC'),
+                CryptoTicker( font='sans'),
```

Comment:
Can we drop this whitespace too or is that hard to do?

Reply:
I honestly have no idea, like I said haven't used bowler and the documentation is a bit sparse.

Reply:
Ok, I think this is reasonable for now; if we decide we want to fix up formatting maybe we should just run flake8 or something at the end. Seems much simpler than wading through bowler, I agree.